src/ImageSharp.Web/FormattedImage.cs

```diff
@@ -97,6 +98,45 @@ public static FormattedImage Load(Configuration configuration, Stream source)
         /// <param name="destination">The destination stream.</param>
         public void Save(Stream destination) => this.Image.Save(destination, this.encoder);
 
+        /// <summary>
+        /// Returns a value indicating whether the source image contains EXIF metadata for <see cref="ExifTag.Orientation"/>
+        /// indicating that the image is rotated (not flipped).
+        /// </summary>
+        /// <param name="orientation">The decoded orientation. Use <see cref="ExifOrientationMode"/> for comparison.</param>
+        /// <returns>The <see cref="bool"/> indicating whether the image contains EXIF metadata indicating that the image is rotated.</returns>
+        public bool IsExifRotated(out ushort orientation)
```

Comment:
I've made this public as it's useful for other processor implementations. The EXIF parsing logic is identical to `AutoOrientProcessor<T>.GetExifOrientation()`

Reply:
Why is this only checking for rotated images and not flipped ones? I think that made sense when only considering width/height, but because the `ResizeWebProcessor` also needs to handle the correct X/Y and anchor position, this should also return `true` when it's flipped.

I'd also rename it to `TryGetExifOrientation(out ushort orientation)` and return whether the image contains a valid EXIF orientation value or not. Then just check if it's false or the orientation is set to 1 (top/left) before doing any of the transforms.

Reply:
Why is this only checking for rotated images and not flipped ones? 

Because rotated is an important metric as that affects Size. Flipped does not affect that property and I wanted a quick way to tell, writing the method to reflect that desire. Can easily refactor though to suit your design and move rotate the check inside the processor itself.